integration-tests/tests/mysql.test.ts

```diff
@@ -1967,3 +1967,52 @@ test.serial('update undefined', async (t) => {
 
 	await db.execute(sql`drop table ${users}`);
 });
+
+test.serial('utc config for datetime', async (t) => {
+	const { db } = t.context;
+
+	await db.execute(sql`drop table if exists \`datestable\``);
+	await db.execute(
+		sql`
+			create table \`datestable\` (
+				\`datetime_utc\` datetime(3),
+				\`datetime\` datetime(3),
+				\`datetime_as_string\` datetime
+			)
+		`,
+	);
+	const datesTable2 = mysqlTable('datestable', {
+		datetimeUTC: datetime('datetime_utc', { fsp: 3, mode: 'date' }),
+		datetime: datetime('datetime', { fsp: 3 }),
+		datetimeAsString: datetime('datetime_as_string', { mode: 'string' }),
+	});
+
+	const dateObj = new Date('2022-11-11');
+	const dateUtc = new Date(Date.now());
+
+	await db.insert(datesTable2).values({
+		datetimeUTC: dateUtc,
+		datetime: dateObj,
+		datetimeAsString: '2022-11-11 12:12:12',
+	});
+
+	const res = await db.select().from(datesTable2);
+
+	const [rawSelect] = await db.execute(sql`select \`datetime_utc\` from \`datestable\``);
+	const datetimeUTC = (rawSelect as unknown as [{ datetime_utc: string }])[0];
+
+	t.is(datetimeUTC.datetime_utc.replace(' ', 'T') + 'Z', dateUtc.toISOString());
```

Comment:
Let's compare `datetime_utc` to a hardcoded string without any transformations. The goal is to verify that the DB value is in an exact format, not that it corresponds to the original date - that is checked in other parts of the test.